Test/Invoke-WhiskeyTask.Tests.ps1

```diff
@@ -388,6 +388,42 @@ function ThenTaskRanWithoutParameter
     }
 }
 
+function ThenTempDirectoryCreated 
+{
+    param(
+        $TaskName
+    )
+
+    $expectedTempPath = Join-Path -Path $context.OutputDirectory -ChildPath ('Temp.{0}.' -f $TaskName)

```

Comment:
I *think* for the regex to be correct you need to escape the `.` character in `Temp.{0}.` as well as any `.` characters in `$TaskName`.

Reply:
On the next line, you'll see I'm using `[regex]::escape` to escape all those unfriendly characters.

Reply:
Ah yes, I skipped over that.